lua/efmls-configs/formatters/eslint.lua

```diff
@@ -0,0 +1,10 @@
+local fs = require('efmls-configs.fs')
+
+local formatter = 'eslint'
+local args = '--fix-to-stdout --stdin-filename ${INPUT} --stdin'
```

Comment:
This doesn't seem to work, it doesn't seem like they have implemented `--fix-to-stdout` yet, it's only a valid eslint_d flag.

This will have to be `--fix ${INPUT}`.